src/scripts/component/homepage.js

```diff
@@ -0,0 +1,16 @@
+// Curt
+
+function populateHomepage () {
+    let homepageContainer = document.querySelector("#container")
+    homepageContainer.innerHTML = `
+    <div id="mainHomepageDiv">
+    <h1>Bitchin. It's a homepage</h1>
+    <section id="eventsSection></section>
```

Comment:
Missing some quotation marks here and below

Reply:
thanks. fixed it